docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -25,11 +25,28 @@ At the top right-hand side of the central pane, click on *Create Kubernetes cl
 A new pane named *Create {{k8s_management_service}} Shoot Cluster* slides over from the right-hand side of the browser.
 Type in a name for the new shoot cluster, and select a region.
 Also, select the version of Kubernetes the new cluster will be running.
-In the example below, we have chosen version 1.28.6, which, at the time of writing, was the latest supported in {{brand}}.
+In the example below, we have chosen version 1.28.7, which, at the time of writing, was the latest supported in {{brand}}.
 
 !["Create {{k8s_management_service}} Shoot Cluster" panel, showing options to type in a cluster name, select region and Kubernetes version](assets/create-shoot-1.png)
 
-Scroll down a bit until you bring the *Worker Groups* section into full view.
+Next, you may accept the proposed CIDR for the worker nodes or type in the one you prefer.
```

Comment:
I believe we need a comma before "or" here, and also I don't think I like the phrase "the CIDR". CIDR is Classless Inter-Domain Routing; the proper term here would be "the CIDR network address" or some such.

Now I understand that the incorrect term is used in CCMP, so this adds potential confusion, but maybe we can get away with something like this:
```suggestion
Next, you may accept the proposed network address (in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation) for the worker nodes, or type in the one you prefer.
```
What do you think?

Reply:
Even though I know what CIDR means, I admit I got carried away by the CCMP wording. So, yes, what you propose is 100% accurate, much betterer (sic), and we should go with that.